zabbix/Makefile

```diff
@@ -10,19 +10,21 @@ SCRIPTS = postinst
 include ../makefiles/common-plugin.mk
 targets = $(addprefix target/, $(FILES))
 
+PYZABBIX_VERSION = 0.7.5
+
 $(targets): target/%: %
 	cp -a $< target/
 
-master.zip:
-	wget https://github.com/adubkov/py-zabbix/archive/1.1.4.zip
+$(PYZABBIX_VERSION).zip:
+	wget https://github.com/lukecyca/pyzabbix/archive/$(PYZABBIX_VERSION).zip
```

Comment:
could you comment this line and download from https://repository.rudder.io/build-dependencies/pyzabbix/0.7.5.zip instead?